site/source/_posts/2019/01/28/Version-5-Dojo.md

```diff
@@ -15,18 +15,29 @@ We're excited to announce the 5.0.0 release of Dojo. This version builds on prev
 
-By using `@dojo/framework/shim` ponyfills for [`WebAnimations`][WebAnimationsAPI], [`IntersectionObserver`][IntersectionObserverAPI], [`ResizeObserver`][ResizeObserverAPI], and [`fetch`][FetchAPI] the Dojo build produces separate "platform" bundles that will be **only** be loaded based on two key conditions:
+The Dojo build produces separate "platform" bundles that will be **only** be loaded based on two key conditions:
 
 1) The shim module is imported somewhere in an application.
 1) A users browser does not natively support the browser feature.
 
 In short, if your application does not use the browser feature or you use a browser with native support, the polyfill will not be loaded! This change means serving less JavaScript and improving your application performance without compromising on features.
 
+### New Polyfills Added to Shim
+
+We've added additional third party polyfills to `@dojo/framework/shim`:
+
+* [`fetch`][FetchAPI] - The Fetch API provides an interface for fetching resources (including across the network).
+* [`IntersectionObserver`][IntersectionObserverAPI] - The Intersection Observer API provides a way to asynchronously observe changes in the intersection of a target element with an ancestor element or with a top-level document's viewport.
+* [`WebAnimations`][WebAnimationsAPI] - The Web Animations API allows for synchronizing and timing changes to the presentation of a Web page, i.e. animation of DOM elements.
+* [`ResizeObserver`][ResizeObserverAPI] - The ResizeObserver interface reports changes to the content rectangle of an Element or the bounding box of an SVGElement. The content rectangle is the box in which content can be placed, meaning the border box minus the padding.
+
+To use these additional polyfills they should be imported like any other ponyfill provided by `@dojo/framework/shim` and they no longer need to be explicitly added/import to your Dojo application.
+
 ### Better Build Time Rendering (BTR)
 
 Build time rendering provides the rendering of your application to HTML during the build and in-lines critical CSS. BTR allows an application to render static HTML pages and offer some advantages of server-side rendering (SSR) such as performance and search engine optimization without the complexities of running a server to support full SSR. Build time rendering has been available via the Dojo [cli-build-app][CliBuildAppReadme]  command since our initial 2.0.0 release.
 
-The 5.0.0 release brings a more robust, flexible and feature-rich build time render by running your application at build time in a browser environment.
+The 5.0.0 release brings a more robust, flexible and feature-rich build time rendering. This is made possible by running your application in a browser environment, ensuring maximum support for browser APIs and functionality.
```

Comment:
This is still a cop out I think sorry 🙈. Lemme try:
We've made a number of stability and feature enhancements in 5.0.0 thanks to now running your application in a real browser environment. New features include:

* Support for StateHistory api (see below)
* Dojo Blocks (see below)
* Multiple page HTML generation
* Screenshots of pages visited during Build Time Rendering
* Better error messaging